source/draw/RasterBgra.ooc

```diff
@@ -171,5 +171,13 @@ RasterBgra: class extends RasterPacked {
 			(top * (left * topLeft alpha + (1 - left) * topRight alpha) + (1 - top) * (left * bottomLeft alpha + (1 - left) * bottomRight alpha))
 		)
 	}
+	swapRedBlue: func {
```

Comment:
Personally, I would prefer a name like `RgbToBgr`, but then it would do the exact same thing as `BgrToRgb` and things would get confusing...